test/dpg/version-tolerant/Expected/AcceptanceTests/DPGServiceDrivenUpdateOneVersionTolerant/dpgservicedrivenupdateoneversiontolerant/aio/operations/_operations.py

```diff
@@ -278,6 +278,11 @@ async def post_parameters(self, parameter: Union[JSON, IO], **kwargs: Any) -> JS
         _content = None
         if isinstance(parameter, (IO, bytes)):
             _content = parameter
+            if not content_type:
+                raise Exception(
```

Comment:
It should be the same error as would happen if you didn't pass in a required param to a function, which is a `TypeError`. You can copy-paste the exact string response that's in the error in this case

Reply:
good point!